code/modules/mob/living/living.dm

```diff
@@ -224,18 +224,35 @@
 /mob/living/resist_grab(moving_resist)
 	if(!pulledby)
 		return
-	if(pulledby.grab_level)
-		if(prob(50))
-			playsound(src.loc, 'sound/weapons/thudswoosh.ogg', 25, 1, 7)
-			visible_message(SPAN_DANGER("[src] has broken free of [pulledby]'s grip!"), null, null, 5)
+	// vars for checks of strengh
+	var/pulledby_is_strong = HAS_TRAIT(pulledby, TRAIT_SUPER_STRONG)
+	var/src_is_strong = HAS_TRAIT(src, TRAIT_SUPER_STRONG)
+
+	// Passive grabs will also be broken, UNLESS pulledby has super strength, but this is ignored if you also have super strength
+	if(pulledby.grab_level || (pulledby_is_strong && !src_is_strong))
+		// Chance for person to break free of grip, defaults to 50.
+		var/chance = 50
```

Comment:
Thank you!

As a last thing, could you flip this so that the `stop_pulling()` part comes first? Since it returns after the proc call you can get an indentation level back for free.
I.e:
```dm
	// Passive grabs will also be broken, UNLESS pulledby has super strength, but this is ignored if you also have super strength
	if(!pulledby.grab_level && (!pulledby_is_strong || src_is_strong))
		pulledby.stop_pulling()
		return TRUE

	// Chance for person to break free of grip, defaults to 50.
	var/chance = 50
	if(src_is_strong && !isxeno(pulledby)) // no extra chance to resist warrior grabs
		chance += 30 // you are strong, you can overpower them easier
etc. etc.
```